### DISCUSSION TITLE
qualcuno ha comprato valvole tado di recente?!

### DISCUSSION THREAD

**Discussion by 50494554524F** [indomus/forum#2882] [Hardware & Firmware]
ho comprato altre valvole e con queste ho problemi ad usare le batterie ricaricabili, mi segna sempre quasi subito scariche...

non vorrei fosse un pacco "difettoso", prima di comprarne altre chiedo...  capita a qualcun altro?! grazie

**Comment by snakuzzo** [comment]
È capitato anche a me su alcune teste con delle Amazon ricaricabili. Ma considera che le mie teste hanno tutte almeno 3 anni.

**Reply by 50494554524F** [reply]
sono proprio le amazon, sulle altre vari marchi... uff son nuove, allora gli faccio fare qualche ciclo carica/scarica spinto nel caso fossero addormentate, sennò mi tocca ricomprarle

**Reply by 50494554524F** [reply]
sto provando le varie batterie che ho, mi sa che le nuove valvole sono più schizzinose o, peggio, ne ho due tre mezze difettose... ora le proverò tutte a giro per non buttare altri soldi

**Reply by 50494554524F** [reply]
niente, le amazon non sono buone per le valvole... sulle vecchie non ho nemmeno potuto provarle perché non ci entrano

comprerò le ikea, che già uso; le altre marche buone hanno aumentato di parecchio i prezzi con la scusa dei casini in giro...

**Comment by effe-7** [comment]
Ciao, io ho le teste Tado e le uso con batterie ricaricabili ma le ho acquistate 2-3 anni fa.
Unica nota che ho notato che alcune batterie ricaricabili (specialmente quelle di fascia bassa) con dispositivi elettronici / smart durano niente il problema aumenta se le pile sono quelle che si ricaricano tramite usb

**Comment by apilone** [comment]
Che ricaricabili sono? Se non ricordo male, le uniche ricaricabili citate da Tado sono le Panasonic Eneloop

**Reply by 50494554524F** [reply]
uso energizer 2000mah 1,50€/pz, ikea 2000mha 1,25/pz, una carica dura quasi sempre per la stagione... se serve uso vecchie varta o beghelli mentre aspetto che si ricarichino

le eneloop sono arrivate a costare 4-4,50€/pz, costi irrecuperabili... magari le amazon 2000mah (le ho pagate solo 60c al black friday) hanno solo bisogno di altri cicli per "attivarsi" o delle valvole sono più sensibili, ma ci vuole troppo tempo :D

**Comment by Rocchi84** [comment]
Le uniche batterie "certificate" da tado sono le eneloop.
Io personalmente  ho messo eneloop su tutte e 12 le teste e i 3 termostati (ne avevo già parecchie fortunatamente) e devo dire che se caricate bene, durano mesi, in alcune stanze, con meno cambi di temperatura, fanno anche la stagione invernale completa.

Costano parecchio, sicuramente, ma se uso batterie non ricaricabili, in uno o 2 anni, ho già speso di più.
Tutte le altre batterie ricaricabili che ho usato negli anni, perdono tantissima capacità (uso caricatore professionale, non roba da supermercato), invece le eneloop sembrano sempre nuove.
Quando le acquisto le ho sempre trovate con capacità superiore ai dati di targa (a volte sfiorano i 2000 quelle da 1900) e anche dopo anni (alcune le ho da piu di 10 anni) hanno perso pochissima capacità.
sinceramente preferisco investire su batterie che mi durano oltre 10 anni, piuttosto che spendere un po meno su altri marchi che sul lungo periodo non mi offrono le stesse garanzie.

io da quando hanno rilasciato il firmware che consentiva l'uso delle batterie ricaricabili, ho passato tutto alle ricaricabili, perché prima scaricavano le battere abbastanza velocemente.
Poi secondo me è stato ottimizzato anche il consumo di corrente, perché inizialmente secondo me erano decisamente energivore.

Comunque se la valvola non riconosce correttamente la batteria (tra alcalina e ricaricabile), potrebbe comportarsi come dici, perché la curva di scarica delle ricaricabili è molto diversa dalle alcaline, e il sintomo è proprio quello che segna subito scarica anche se poi va avanti a lavorare magari per settimane o mesi.
Potresti provare con 2 eneloop e ti togli il dubbio.

**Reply by SaintTDI** [reply]
Ciao!
Stavo pensando di passare alle eneloop... perchè in effetti il consumo di batterie alcaline è esagerato con le valvole Tado... tra l'altro all'ambiente non è che fa cosi bene.
Leggo che usi un caricatore professionale, non è che potresti dirmi quale? Cosi almeno parto con un setup collaudato :) grazie tante!

**Reply by effe-7** [reply]
Io uso batterie ricaricabili Varta e quando le ho acquistate avevo trovato una promo dove era incluso il carica pile.
E devo dire che mi trovo benissimo.
Nelle valvole Tado le sostituisco annualmente.

**Comment by Rocchi84** [comment]
> Leggo che usi un caricatore professionale, non è che potresti dirmi quale? Cosi almeno parto con un setup collaudato :) grazie tante!

Io uso questo: Maha MH-C9000
Ce ne sono vari in commercio di questo tipo, l'unico "problema" è che devi sapere come trattare le varie batterie in base ai loro parametri specifici.
Suggerimento mio: se decidi di usare solo eneloop, allora potresti comprare il bundle (batterie e carica) direttamente di eneloop, così sei sicuro di non sbagliare parametri e trattarle sempre al meglio.

Ti consiglio di dare un occhiata questo sito che io definisco "la bibbia delle eneloop" [https://eneloop101.com/](url)
qui trovi i codici di produzione dei vari anni, cosi eviti di comprare batterie già "vecchie" o versioni obsolete e trovi anche le informazioni sui caricabatterie, versioni fake, ecc..ecc..

Altro consiglio (ma se leggi sul sito che ti ho detto trovi la spiegazione) per le valvole tado non ha senso acquistare le versioni eneloop PRO, perché costano un rene e sono adatte per altri scopi (applicazioni a consumo elevato)

**Reply by SaintTDI** [reply]
perfetto! ti ringrazio tanto ora mi studio il sito che mi hai passato ;)
